asmcli/commands/experimental/vm.sh

```diff
@@ -321,7 +321,7 @@ is_google_identity_provider_installed() {
 }
 
 is_idp_crd_installed() {
-  if ! kubectl api-resources --api-group=security.cloud.google.com | grep -q identityproviders; then
+  if ! retry 2 kubectl api-resources --api-group=security.cloud.google.com | grep -q identityproviders; then
```

Comment:
If we use `kubectl get crd`, can we potentially avoid the error?

Also do we need a retry on the entirety of cluster registration verification?

Reply:
Updated to `kubectl get crd`.  Clay advised against retry the entirety so I am leaving as is.